### ISSUE TITLE
49708 DAYS =136 YEARS BLOCKED TO CHAT 

### ISSUE THREAD

**Issue by onesoull** [dalaranwow/dalaran-wow#5183]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)

**Expected behaviour**: Tell us what should happen instead

**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
MY name alt is onesoul i got banned from the chat for 136 years it all happen because off my f@@king cousin who spammed global i got blocked by user amanthhul with a ban off 49708 days =136 years i apologize off all the things my dumm cousin did at that time it won t happen again all i want to say its i can t talk in the dungeons and i get kicked for not respecting them by answering back if you can t lift my ban please tell me what steps to take to be able do that ? cuss i don t want to lose my DEATH KNIGHT it means a lot to me if it s possible i want to be able to chat if in worst case scenario the only way i can lift the ban is 136 years all i can say i will not delete my alt and wait until than even if i am the only guy alive and i will keep dalaran server for that long and even if i become a sponsor off the dalaran after 10 years and hold the  server and have the right as game master to lift my alt onesoul ban i wouldn t as respect for the game master Amanthhul !

**Comment by MalkaviaDW** [comment]
Hello Onesoul, 

It should be already solved.

Cheers!
